vue-frontend/src/components/mobile-app-development/Portfolio.vue

```diff
@@ -0,0 +1,244 @@
+          alt="app-image"
+          class="tw-mt-8 tw-h-full tw-w-full tw-object-contain"
+        />
+      </div>
+    </div>
+    <!-- MOBILE UI END -->
+
+    <!-- DESKTOP UI START -->
+    <div class="tw-container tw-hidden md:tw-block">
+      <p
+        class="tw-text-center tw-font-inter-bold tw-text-[3.125rem] tw-leading-[3.438rem] tw-text-white xl:tw-text-[4.125rem] xl:tw-leading-[4.95rem]"
+      >
+        Our portfolios
+      </p>
+      <p
+        class="v2-canopas-gradient-text tw-mt-1 tw-text-center tw-font-inter-medium tw-text-[1.25rem] tw-leading-[1.5rem] xl:tw-text-[1.5rem] xl:tw-leading-[1.8rem]"
+      >
+        1M+ users use our apps everyday
+      </p>
+      <p
+        class="tw-mt-6 tw-text-center tw-font-inter-regular tw-text-[1.25rem] tw-leading-[1.5rem] tw-leading-[1.8rem] tw-text-white/[0.85] xl:tw-text-[1.5rem]"
+      >
+        Discover how we transformed the vision of our clients into successful
+        mobile applications.
+      </p>
+      <div
+        class="tw-mt-32 tw-flex tw-flex-col"
+        v-for="(portfolio, index) in portfolios"
+        :key="index"
+      >
+        <div
+          class="tw-flex tw-flex-row tw-gap-6 lg:tw-gap-14"
+          :class="
+            portfolio.id == 2 || portfolio.id == 4 ? 'tw-flex-row-reverse' : ''
+          "
+        >
+          <div
+            class="tw-flex tw-flex-[61%] tw-flex-col tw-justify-center 2xl:tw-w-[70%]"
+            :class="
+              portfolio.id == 2 || portfolio.id == 4
+                ? 'lg:tw-pl-[4.5rem] 2xl:tw-pl-[6.5rem]'
+                : ''
+            "
+          >
+            <div class="tw-flex tw-flex-row tw-items-center tw-gap-2">
+              <img
+                :src="
+                  portfolio.id == 2 || portfolio.id == 3
+                    ? portfolio.desktopLogo
+                    : portfolio.logo
+                "
+                alt="app-logo"
+                class="tw-h-[30px] tw-w-[30px] tw-object-contain"
+                :class="index == 3 ? 'tw-w-[75px]' : ''"
+              />
+              <p
+                class="tw-font-inter-semibold tw-text-[1.25rem] tw-leading-[1.513rem]"
+                :class="
+                  portfolio.id == 1
+                    ? 'tw-from-[#26AEF9] tw-to-[#2671FA] tw-bg-clip-text tw-text-transparent tw-bg-gradient-[180deg]'
+                    : 'tw-text-white '
+                "
+              >
+                {{ portfolio.name }}
+              </p>
+            </div>
+            <p
+              class="tw-mt-2 tw-w-[90%] tw-font-inter-semibold tw-text-[2.063rem] tw-leading-[2.5rem] tw-text-white xl:tw-text-[3rem] xl:tw-leading-[3.631rem]"
+              :class="
+                portfolio.id == 1 || portfolio.id == 3
+                  ? 'lg:tw-w-[67%] xl:tw-w-[81%] 2xl:tw-w-[83%]'
+                  : 'lg:tw-w-[78%] xl:tw-w-[89%] 2xl:tw-w-[82%]'
+              "
+            >
+              {{ portfolio.title }}
```

Comment:
Let's add click events on the title, logo and image on desktop and on image in mobile.

Add hover effect on title: underline should be display on hover from left to right